Ease listing contributors (#135)
OpenTermsArchive/docs: CONTRIBUTING.md

# Contributing to this repository

## 🚀 How to Contribute

> - You can check the open issues in this repository, assign yourself and solve them.

Reviewer: Unfortunately assigning has to be done by an admin, people can't self-assign.

```suggestion
- You can check the open issues in this repository, comment to state your interest and solve them.
```

> - You can also make suggestions for additions or modifications of this documentation website.

Reviewer: ```suggestion
- You can also open issues to suggest additions or changes to this documentation website.
```

## 📜 Read our Code of Conduct

> Before you begin coding and collaborating, please read our [Code of Conduct](./CODE_OF_CONDUCT.md) thoroughly to understand the standards (that you are required to adhere to) for community engagement. As part of our open-source community, we hold ourselves and other contributors to a high standard of communication. As a participant and contributor to this project, you are agreeing to abide by our Code of Conduct.

Reviewer: ```suggestion
Before you begin coding and collaborating, please read our [Code of Conduct](./CODE_OF_CONDUCT.md) to understand the standards (that you are required to adhere to) for community engagement. As part of our open-source community, we hold ourselves and other contributors to a high standard of communication. As a participant and contributor to this project, you are agreeing to abide by our Code of Conduct.
```

# Contributing to this repository

## Contributions to make

You can add documents for tracking by submitting a pull request either using the GUI contributing tool, or by creating a JSON file. There are three kind of contributions you can make:

- Adding declarations to a collection;
- Updating declarations in a collection;

> - Removing declarations from a collection.

Reviewer: This doesn't seem appropriate for the `docs` repository 😅
